Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -30,7 +31,7 @@
 def chunks(l, n):
     """Yield successive n-sized chunks from l."""
     for i in range(0, len(l), n):
-        yield l[i:i + n]
+        yield l[i : i + n]
```

Comment:
It would be very hard for me to comply with our current Flake8 checks (especially in regards to imports) without using code autoformatting. Sorry about that.
If we had introduced code autoformatting along with introducing the checks, this problem would have never arisen.

Reply:
Yeah we discussed this already yesterday. Although I must say that most of these cosmetic changes indeed improve readability 👍